### ISSUE TITLE
Get familiar with software/turtlebot

### ISSUE THREAD

**Issue by JonasPetersenFAU** [amosproj/amos2022ss03-turtlebot-fleet-management#3]
## User story
1. As a SD
2. I want / need to get familiar with software, robot and user interface
3. So that i can start with the project

## Acceptance criteria
* Get familiar with software and user interface
* Try out robot for fun

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
